core-bundle/tests/Fixtures/Util/LegacySimpleTokenParser.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Tests\Fixtures\Util;
+
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\LogLevel;
+
+class LegacySimpleTokenParser implements LoggerAwareInterface
+{
+    use LoggerAwareTrait;
+
+    /**
+     * This is a copy of the StringUtil::parseSimpleTokens implementation prior
```

Comment:
Can just use `System::setContainer()` no? I find it quite confusing to have this class in our testsuite :D

Reply:
I did not intend to keep this anyways. But it was an easy way to compare both implementations to find out what should be done.

Reply:
Ah and maybe the docbloc is misleading. This is the code from the 4.9 codebase that differs quite a bit from the current one.